File: codeforces/69/A.pas
var
 a,m,j,i,n,g,sumx,sumy,sumz : longint;
 tabX : array[1..200] of integer;
 tabY : array[1..200] of integer;
 tabZ : array[1..200] of integer;
begin
 readln(n);
 for i:= 1 to n do
  readln(tabX[i],tabY[i],tabZ[i]);
 for i:= 1 to n do
  begin
   sumx:=sumx+tabX[i];
   sumy:=sumy+tabY[i];
   sumz:=sumz+tabZ[i];
  end;
 if (sumx=0) and (sumy=0) and (sumz=0) then writeln('YES') else writeln('NO');
end.

File: codeforces/69/A.cpp
#include <iostream>
using namespace std;

int main()
{
	int i,j,k,n,sumX,sumY,sumZ;
	cin>>n;
	sumX=0;
	sumY=0;
	sumZ=0;
	int a,b,c;
	for (i=1;i<=n;i++)
	{
		cin>>a>>b>>c;
		sumX+=a;
		sumY+=b;
		sumZ+=c;
	}
	if (!sumX && !sumY && !sumZ)
	 cout<<"YES\n";
	else
	 cout<<"NO\n";
}
